src/common/tracking/amplitude.js

```diff
@@ -1,48 +1,33 @@
-import amplitude from "amplitude-js";
+import * as amplitude from "@amplitude/analytics-browser";
 import getSessionId from "../../session";
 
-export function initAmplitude() {
-    amplitude.getInstance();
-    amplitude.init(window.__AMPLITUDE_TOKEN__, null, {
-        apiEndpoint: "amplitude.nav.no/collect",
-        batchEvents: false,
-        includeReferrer: true,
-        includeUtm: true,
-        saveEvents: false,
-        transport: "beacon"
-    });
+function getAmplitudeKey() {
+    if (window.location.href.includes("nav.no")) return window.__AMPLITUDE_TOKEN__;
```

Comment:
pirk: hadde vurdert å sette __AMPLITUDE_TOKEN__ til `""` som default verdi og bare kalt 
```return window.__AMPLITUDE_TOKEN__```

Reply:
Hvor tenkte du på her? Afaik er det bare en env variabel som hentes fra naisrator-dev/-prod eller runDev (lokalt). Så isåfall i `runDev.sh`?

Reply:
Hmm interessant, som den er nå så ville den i `runDev.sh` aldri blitt brukt fra det jeg ser da den kjøres ikke i dev eller prod. 

I det tilfelle ville jeg heller vurdert å fjerne AMPLITUDE_TOKEN fra `runDev.sh` og så håndtere `undefined` (altså når AMPLITUDE_TOKEN ikke eksisterer). Fra det jeg ser så "tror" jeg at koden allerede håndterer `undefined` men hadde vel dobbelt sjekket da jeg er usikker på hva `!undefined` betyr.